test/smoke_web/dapi.js

```diff
@@ -123,7 +123,8 @@ describe('DAPI', () => {
           }
 
           expect(dataContract[hostName]).to.be.undefined();
-          expect(dataContractError[hostName].code).to.be.equal(5);
```

Comment:
Let's not remove it for now. Just comment out please. We will bring this back at some point

Reply:
It was removed in #550 so let's just stay consistent with that?

Reply:
Oh.. that's not great. Could you please recover that one as well and put todos to bring this back?